### ISSUE TITLE
Using swifter as an option

### ISSUE THREAD

**Issue by reza1615** [man-group/dtale#295]
It would be nice to have swifter as an option before loading dtale. 

-  It can pass a global variable to whole part of the dtale codes. If that variable is true > import swifter and add it before apply.
 - or We can do monkey patch on apply () if is_swifter==True .
https://stackoverflow.com/a/47939992/5833945
https://github.com/johnson-li/fast_pandas

**Comment by aschonfeld** [comment]
@reza1615 I'll be including this in my release.  The way I approached it was to simply use this function anytime I was using `apply`:
```python
def apply(df, func, *args, **kwargs):
    try:
        import swifter

        return df.swifter.progress_bar(False).apply(func, *args, **kwargs)
    except ImportError:
        return df.apply(func, *args, **kwargs)
```
So essentially, if you install `swifter` it will be used, otherwise the code executes as it currently does.

**Comment by reza1615** [comment]
great idea. in documentation we can mention it

**Comment by aschonfeld** [comment]
Yep here's the [documentation](https://github.com/man-group/dtale/tree/bugfixes_20201122#using-swifter) I had come up with

**Comment by aschonfeld** [comment]
released in v1.24.0
